### PR TITLE
Created EmbeddedHelp.txt

### PR DESCRIPTION
**Pull request by Tominous**
**Cleaner and newest version of the current help command!**

### CODE REVIEW

**Comment by Swordington** [comment]
Hi there and thanks for submitting a command! What would be a possible use for this command and how is it different from other commands already in DCC's repo? Also, when would a user use this command?

**Comment by Swordington** [comment]
Hi there! If you don't respond within the next 24 hours, we'll assume you no longer want this PR merged.

File: EmbeddedHelp.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{!announce {channel} The prefix for **{server.name}** is `{prefix}`
+You can find more information at https://dyno.gg/
+_　_
+**ADDITIONAL LINKS AND INFORMATION/HELP**
+_　_
+[Commands](https://wwwdynobot.net/commands) **–** Gives you a detailed list of commands.
+[Manage Server](https://dyno.gg/manage/{server.id}) **–** Requires `Administrator` or `Manage Server`.
+[Dyno Discord](https://discord.gg/Dyno) **–** Dyno’s Official Support Server
+[Get Dyno](https://discordapp.com/oauth2/authorize?client_id=161660517914509312&scope=bot%20identify%20email%20guilds%20guilds.join%20messages.read%20connections&response_type=code&redirect_uri=https://www.dynobot.net/return&permissions=2146958847) **–** Invites Dyno into your server!
+[Donate / Get Premium](https://dyno.gg/upgrade) **–** Unlocks the most awesome perks! Type `?premium`!
```
Comment (no-egrets) [inline EmbeddedHelp.txt:10]: If this CC _is_ added to the repo, the prefix here should be dynamic:
```suggestion
[Donate / Get Premium](https://dyno.gg/upgrade) **–** Unlocks the most awesome perks! Type `{prefix}premium`!
```

**Comment by Swordington** [comment]
This PR is being denied due to:
**No response from PR submitter.**

**Pull request closed without merging**